packages/@aws-cdk/aws-codepipeline-actions/lib/cloudformation/stackset-action.ts

```diff
@@ -0,0 +1,691 @@
+      configuration: {
+        StackSetName: this.props.stackSetName,
+        Description: this.props.description,
+        TemplatePath: this.props.templatePath.location,
+        Parameters: this.props.parameters?.render(),
+        Capabilities: parseCapabilities(this.props.cfnCapabilities),
+        FailureTolerancePercentage: this.props.failureTolerancePercentage,
+        MaxConcurrentPercentage: this.props.maxConcurrentPercentage,
+        ...instancesResult?.stackSetConfiguration,
+        ...permissionModelBind?.stackSetConfiguration,
+      },
+    };
+  }
+}
+
+/**
+ * Properties for the CloudFormationStackInstancesAction
+ */
+export interface CloudFormationStackInstancesActionProps extends codepipeline.CommonAwsActionProps {
+  /**
+   * The AWS region the given Action resides in.
+   * Note that a cross-region Pipeline requires replication buckets to function correctly.
+   * You can provide their names with the {@link PipelineProps#crossRegionReplicationBuckets} property.
+   * If you don't, the CodePipeline Construct will create new Stacks in your CDK app containing those buckets,
+   * that you will need to `cdk deploy` before deploying the main, Pipeline-containing Stack.
+   *
+   * @default - the Action resides in the same region as the Pipeline
+   */
+  readonly region?: string;
+
+  /**
+   * The name of the StackSet we are adding instances to
+   */
+  readonly stackSetName: string;
+
+  /**
+   * Specify where to create or update Stack Instances
+   *
+   * You can specify either AWS Accounts Ids or AWS Organizations Organizational Units.
+   */
+  readonly stackInstances: StackInstances;
+
+  /**
+   * Parameter values that only apply to the current Stack Instances
+   *
+   * These parameters are shared between all instances added by this action.
+   *
+   * @default - No parameters will be overridden.
+   */
+  readonly parameterOverrides?: StackSetParameters;
+
+  /**
+   * The percentage of accounts per Region for which this stack operation can fail before AWS CloudFormation stops the operation in that Region. If
+   * the operation is stopped in a Region, AWS CloudFormation doesn't attempt the operation in subsequent Regions. When calculating the number
+   * of accounts based on the specified percentage, AWS CloudFormation rounds down to the next whole number.
+   *
+   * @default 0
+   */
+  readonly failureTolerancePercentage?: number;
+
+  /**
+   * The maximum percentage of accounts in which to perform this operation at one time. When calculating the number of accounts based on the specified
+   * percentage, AWS CloudFormation rounds down to the next whole number. If rounding down would result in zero, AWS CloudFormation sets the number as
+   * one instead. Although you use this setting to specify the maximum, for large deployments the actual number of accounts acted upon concurrently
+   * may be lower due to service throttling.
+   *
+   * @default 1
+   */
+  readonly maxConcurrentPercentage?: number;
```

Comment:
Do we need to extract a super interface for `CloudFormationStackInstancesActionProps` and `CloudFormationStackSetActionProps`? These are the same in both of them I belive.

Reply:
They are. It's not really worth it for these couple properties (and the rest need slightly different docstrings).

Reply:
Can you do it anyway? For me? ❤️